src/components/line-chart-audification/line-chart-audification.component.spec.ts

```diff
@@ -21,26 +22,31 @@ describe('LineChartAudificationComponent', () => {
   beforeEach(() => {
     TestBed.configureTestingModule({
       imports: [
-        DataModule,
+        MatCardModule,
       ],
       declarations: [
         LineChartComponent,
       ],
     });
     const hostFixture = TestBed.createComponent(LineChartComponent);
     const host = hostFixture.componentInstance;
-    host.measureName = 'activeUsers';
-    host.ngOnChanges({ measureName: new SimpleChange(null, host.measureName, true) });
+    const time = new Date();
+    host.meta = createLineChartMeta(
+      'Title',
+      () => [{
```

Comment:
Could we define this callback separately and reference it here? It will make it easier to know what it does/ is for.

Reply:
Moved to #54 . Will be resolved in separate PRs.